core/extension/gdnative_interface.h

```diff
@@ -449,6 +449,8 @@ typedef enum {
 	GDNATIVE_INITIALIZATION_SERVERS,
 	GDNATIVE_INITIALIZATION_SCENE,
 	GDNATIVE_INITIALIZATION_EDITOR,
+	GDNATIVE_INITIALIZATION_DRIVER,
+	GDNATIVE_MAX_INITIALIZATION_LEVEL,
 } GDNativeInitializationLevel;
```

Comment:
I've added in a MAX entry at the end so we can implement loops in places where that makes sense.

Reply:
Usually we name such `_MAX` entries as a suffix, i.e. `GDNATIVE_INITIALIZATION_MAX`. It can lead to tricky situations when `MAX` is actually an actually used enum value (math functions), but it makes the prefix consistent in the docs (though that's not exposed so might not matter much).